fiorella/src/styles/layout/_header.scss

```diff
@@ -0,0 +1,247 @@
+/*================ Site Header ================*/
+.homepage-logo {
+  display: none;
+}
+
+.desktop-btn-search {
+  display: none;
+}
+
+.header-buttons-container {
+  display: none;
+}
+
+.menu-container {
+  background-color: $color-white-solid;
+  font-family: $font-menu-header;
+  height: 100%;
+  width: 0;
+  overflow-x: hidden;
+  position: fixed;
+  transition: 0.5s;
+  z-index: 2;
+
+  a {
+    color: $color-black-solid;
+    text-decoration: none;
+    font-size: 25px;
+  }
+
+  @include reset-ul(block);
+}
+
+.show-menu {
+  width: 100%;
+  padding: 0 20px ;
+}
+
+.menu-top-section {
+  display: flex;
+  justify-content: center;
+  padding-top: 10px;
+  position: relative;
+
+  .menu-close {
+    @extend %positioning;
+    justify-content: flex-end;
+  }
+
+  button {
+    background-color: transparent;
+    border-style: none;
+  }
+}
+
+.search-container {
+  border-bottom-style: solid;
+  border-bottom-color: $color-featured;
+  margin: 20px 0 5px 0;
+  padding-bottom: 5px;
+
+  form {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .desktop-btn-search {
+    display: none;
+  }
+
+  button {
+    @extend %transparent-button;
+    padding-right: 0;
+  }
+
+  input {
+    background-color: transparent;
+    border-style: none;
+    font-size: 15px;
+    text-transform: uppercase;
+  }
+
+  ::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
+    color: $color-black-solid;
+    opacity: 1; /* Firefox */
+  }
+
+  :-ms-input-placeholder { /* Internet Explorer 10-11 */
+    color: $color-black-solid;
+  }
+
+  ::-ms-input-placeholder { /* Microsoft Edge */
+    color: $color-black-solid;
+  }
+}
+
+.mobile-nav {
+  border-bottom-style: solid;
+  border-bottom-color: $color-black-solid;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 10px;
+  margin-bottom: 10px;
+
+  a {
+    text-transform: uppercase;
+  }
+}
+
+.mobile-user-buttons {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+
+  a {
+    color: $color-featured;
+  }
+}
+
+.mobile-info-buttons {
+  margin-top: 10px;
+}
+
+.mobile-top-section {
+  display: flex;
+  align-items: center;
+  background-color: $color-featured;
+  height: 40px;
+  justify-content: center;
+  position: relative;
+
+  button {
+    @extend %transparent-button;
+  }
+
+  p {
+    background-color: $color-featured;
+    border-style: none;
+    border-radius: 50%;
+    color: $color-white-solid;
+    display: inline;
+    padding: 4px 10px;
+  }
+
+  .header-card-button {
+    @extend %positioning;
+    justify-content: flex-end;
+    padding-right: 10px;
+
+    p {
+      display: inline;
+      font-size: 14px;
+      margin: 0;
+    }
+  }
+
+  img {
+    width: 100%;
+  }
+}
+
+.home-screen-mobile-search {
+  margin: 10px 8px 0 8px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `10px 8px 0` instead of `10px 8px 0 8px`